pyrasterframes/src/main/python/docs/ipython.pymd

```diff
@@ -0,0 +1,70 @@
+# IPython/Jupyter Extensions
```

Comment:
i think you need this?

```suggestion
```python, setup, echo=False
from IPython.core.display import display
from pyrasterframes.utils import create_rf_spark_session
from pyrasterframes.rasterfunctions import *

spark = create_rf_spark_session()
```

# IPython/Jupyter Extensions
```